client/utils/events.ts

```diff
@@ -461,14 +464,73 @@ function canMarkEventAsComplete(
     }
 }
 
+/**
+ * Will return true if there is at least one event that can be added
+ */
+export function canAddSomeEventsAsRelatedToPlanningEditor(eventIds: Array<IEventItem['_id']>): boolean {
+    const editor = planningApi.editor(EDITOR_TYPE.INLINE);
```

Comment:
This is where we would use that `event_link_method` config to add additional clause if this action should be available (i.e. config is `one_primary` and currentRelatedEvents is not empty, then return false)